api/types/accesslist/convert/v1/member.go

```diff
@@ -37,13 +37,13 @@ func FromMemberProto(msg *accesslistv1.Member, opts ...MemberOption) (*accesslis
 		return nil, trace.BadParameter("spec is missing")
 	}
 
-	member, err := accesslist.NewAccessListMember(headerv1.FromMetadataProto(msg.Header.Metadata), accesslist.AccessListMemberSpec{
-		AccessList: msg.Spec.AccessList,
-		Name:       msg.Spec.Name,
-		Joined:     msg.Spec.Joined.AsTime(),
-		Expires:    msg.Spec.Expires.AsTime(),
-		Reason:     msg.Spec.Reason,
-		AddedBy:    msg.Spec.AddedBy,
+	member, err := accesslist.NewAccessListMember(headerv1.FromMetadataProto(msg.GetHeader().GetMetadata()), accesslist.AccessListMemberSpec{
+		AccessList: msg.GetSpec().GetAccessList(),
+		Name:       msg.GetSpec().GetName(),
+		Joined:     msg.GetSpec().GetJoined().AsTime(),
```

Comment:
Proto time is represented as timestamp so this will generate a date with unix timestamp 0 = 1970-01-01.. and so time.IsZero will never return true. Should we add a check like:

```
var t time.Time 
if msg.GetSpec().GetJoined()!=nil{
t = msg.GetSpec().GetJoined().AsTime()
}
```

Reply:
This seems to be  legally behavior where before the `Joined: msg.Spec.Joined.AsTime(),` was used. 

Itself this case is validated by: 
```
func (a *AccessListMember) CheckAndSetDefaults() error {
	if a.Spec.Joined.IsZero() || a.Spec.Joined.Unix() == 0 {
		return trace.BadParameter("member %s: joined field empty or missing", a.Spec.Name)
}
```

https://github.com/gravitational/teleport/blob/master/api/types/accesslist/member.go#L105